groupAndSum.js

```diff
@@ -1,19 +1,62 @@
-// Author Tony Germano 
-// Discusion orginated on Mirth Slack from discussion started by nafwa03
+/**
+ * Author Tony Germano
```

Comment:
His Grandness, Tony Germano

use the mans title please

Reply:
@jonbartels Please do a pull request for suggested changes.